.github/workflows/ci-windows.yml

```diff
@@ -7,7 +7,47 @@ permissions:
 
 jobs:
   Windows:
-    name: Windows
+    name: Windows (Native OPAM)
+    runs-on: windows-latest
+    steps:
+      - name: 🔭 Checkout code
+        uses: actions/checkout@v4
+
+      - name: 😈 Evil hacks
+        run: rm rocq-prover.opam
+
+      - name: 😈😈 Evil hack 2
+        run: rm test-suite/success/rewrule.v
+
+      - name: 😈😈😈 Evil hack 3
+        run: rm test-suite/failure/uip_loop.v
+
+      - name: 😈😈😈😈 Evil hack 4
+        run: rm rocq-doc.opam
+
+      - name: 🐫 Setup OCaml
+        uses: ocaml/setup-ocaml@v3
+        with:
+          ocaml-compiler: 4.14
+          dune-cache: true
+          opam-local-packages: |
+            coqide-server.opam
+            rocq-*.opam
```

Comment:
this is used only for `opam pin --no-action` AFAICT so it may be fine to use `*.opam` (the default value) here

Reply:
I think I had to add this as otherwise the dependencies of rocqide would be installed.